automation_tools/satellite6/upgrade/capsule.py

```diff
@@ -43,7 +43,9 @@ def satellite6_capsule_setup(sat_host, os_version):
             print('The following environment variable(s) must be '
                   'set: {0}.'.format(', '.join(missing_vars)))
             sys.exit(1)
-        cap_image = os.environ.get('RHEV_CAP_IMAGE')
+        os.environ['RHEV_CAP_IMAGE'] = 'qe-capsule-upgrade-rhel6-fixed-cap{}'
+        cap_image = os.environ.get('RHEV_CAP_IMAGE').format(os.environ.get(
+            'FROM_VERSION').replace('.', ''))
```

Comment:
using os.environ.get('SOMEENV') and chaining a string method on it will fail if the env var is not defined:

```
In [5]: os.environ.get('NOEXIST').format(1)
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-5-fd076127bd25> in <module>()
----> 1 os.environ.get('NONEXIST').format(1)

AttributeError: 'NoneType' object has no attribute 'format'
```